.travis/custom_release.sh

```diff
@@ -4,8 +4,7 @@ set -x
 
 if [ "${TRAVIS_BRANCH}" = "master" ]; then
     .travis/release.sh "ci-beta"
-    rm -rf .git
-    .travis/release.sh "ci-stable"
+    git push origin ci-beta:ci-stable
```

Comment:
what is origin in this case?

Reply:
The release script sets all that up -- for context, this is what the scripts are doing prior to this step:

```
+git config --global user.email
travis@example.org
+git ls-remote --exit-code git@github.com:RedHatInsights/drift-frontend-build ci-beta
+git clone --bare --branch ci-beta --depth 1 git@github.com:RedHatInsights/drift-frontend-build .git
Cloning into bare repository '.git'...
+git config core.bare false
+git add -A
+git commit -m 'Cleanup .git before release to ci-stable'
[ci-beta 830af92] Cleanup .git before release to ci-stable
 1 file changed, 3 insertions(+), 3 deletions(-)
+git push origin ci-beta
```

Reply:
i didnt realize pushes were already happening, change looks good